api/payara-bom/pom.xml

```diff
@@ -472,7 +472,7 @@
             <dependency>
                 <groupId>org.glassfish</groupId>
                 <artifactId>jakarta.enterprise.concurrent</artifactId>
-                <version>${concurrent.version}</version>
+                <version>${jakarta.enterprise.concurrent}</version>
```

Comment:
By my suggested "use project names" convention this should probably just be `concurro.version`